diff --git a/Gemfile.plugins.template b/Gemfile.plugins.template
index da2da4d..35c1545 100644
--- a/Gemfile.plugins.template
+++ b/Gemfile.plugins.template
@@ -12,10 +12,10 @@ gem 'dradis-calculator_dread', '~> 3.6', github: 'dradis/dradis-calculator_dread
 # ---------------------------------------------------------------------- Export
 gem 'dradis-csv',         '~> 3.6', github: 'dradis/dradis-csv'
 gem 'dradis-html_export', '~> 3.6', github: 'dradis/dradis-html_export'
-
+# gem 'dradis-pdf_export',            github: 'evs-ch/dradis-pdf_export'
 
 # ---------------------------------------------------------------------- Import
-# gem 'dradis-mediawiki', '~> 3.6', github: 'dradis/dradis-mediawiki'
+gem 'dradis-mediawiki', '~> 3.6', github: 'dradis/dradis-mediawiki'
 # gem 'dradis-vulndb',    '~> 3.6', github: 'dradis/dradis-vulndb'
 
 # ---------------------------------------------------------------------- Upload
diff --git a/config/application.rb b/config/application.rb
index 79d17d7..46fd082 100644
--- a/config/application.rb
+++ b/config/application.rb
@@ -27,6 +27,12 @@ module Dradis
       lib
     ).map { |path| "#{config.root}/#{path}" }
 
+    # this is probably a horrible thing to do, but it _does_ fix this error..:
+    # ActionView::Template::Error (uninitialized constant HTML::Pipeline::DradisFieldableFilter)
+    # In my defense: Stackoverflow made me do it!!11
+    # http://stackoverflow.com/questions/39425319
+    config.eager_load_paths += %W(#{config.root}/lib/)
+
     # Fix relative URLs for mounted engines.
     # See:
     #   https://github.com/activeadmin/activeadmin/issues/101#issuecomment-22273869
diff --git a/config/environments/production.rb b/config/environments/production.rb
index 6b4eb4c..f8713a9 100644
--- a/config/environments/production.rb
+++ b/config/environments/production.rb
@@ -41,7 +41,7 @@ Rails.application.configure do
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
+  config.force_ssl = false
 
   # Use the lowest log level to ensure availability of diagnostic information
   # when problems arise.
